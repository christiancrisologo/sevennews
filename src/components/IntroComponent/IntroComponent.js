import React from "react";
import styled from "styled-components";
import tile from "./assets/tile.jpg";
import logo from "./assets/logo.png";

const Wrapper = styled.section`
  background: #000;
  width: 600px;
  height: 355px;
  background-image: url(${tile});
  margin: 10% auto;
`;

const TitleContainer = styled.div`
  background: rgba(0, 0, 0, 0.7);
  position: relative;
  padding: 0px 10px;
  top: 250px;
  left: 0px;
  width: 100%;
  height: 105px;
  display: flex;
  align-items: center;
`;

const Logo = styled.img`
  width: 80px;
  height: 80px;
`;

const Title = styled.h1`
  font-size: 2.1em;
  text-align: left;
  color: #fff;
  padding-left: 10px;
`;

const IntroComponent = props => (
  <Wrapper>
    <TitleContainer>
      <Logo src={logo} alt="7 news logo" />
      <Title>Home and Away</Title>
    </TitleContainer>
  </Wrapper>
);

export default IntroComponent;
