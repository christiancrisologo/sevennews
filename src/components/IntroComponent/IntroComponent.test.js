import React from "react";
import { shallow } from "enzyme";
import IntroComponent from "./IntroComponent";
import styled from "styled-components";
import renderer from "react-test-renderer";

describe("IntroComponent Test", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<IntroComponent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
