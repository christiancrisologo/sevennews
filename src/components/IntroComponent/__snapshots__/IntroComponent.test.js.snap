// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IntroComponent Test renders correctly 1`] = `
<section
  className="sc-bdVaJa yfbNE"
>
  <div
    className="sc-bwzfXH bDyzcq"
  >
    <img
      alt="7 news logo"
      className="sc-htpNat eZHfRK"
      src="logo.png"
    />
    <h1
      className="sc-bxivhb gRZAkT"
    >
      Home and Away
    </h1>
  </div>
</section>
`;
